src/sources/soundsourceffmpeg.cpp

```diff
@@ -968,9 +968,9 @@ const CSAMPLE* SoundSourceFFmpeg::resampleDecodedAVFrame() {
 #if VERBOSE_DEBUG_LOG
         avTrace("Received resampled frame", *m_pavResampledFrame);
 #endif
-        DEBUG_ASSERT(m_pavDecodedFrame->pts = m_pavResampledFrame->pts);
-        DEBUG_ASSERT(m_pavDecodedFrame->nb_samples =
-                             m_pavResampledFrame->nb_samples);
+        DEBUG_ASSERT(m_pavDecodedFrame->pts == m_pavResampledFrame->pts);
```

Comment:
Ouch!

Reply:
The first assertion seems to be wrong and too restrictive. Presentation time stamps should not matter for resampling.

Reply:
Shall I remove it then?

Reply:
Strange that only the coverage CI job fails because of this.

Reply:
https://github.com/Holzhaus/mixxx/pull/10